### PR TITLE
Use nteract/spawnteract

### PR DESCRIPTION
**Pull request by lgeiger**
This can be merged when `spawnteract 2.1.0` is released.

### CODE REVIEW

File: lib/zmq-kernel.coffee [hunk 1]
```diff
@@ -373,11 +368,11 @@ class ZMQKernel extends Kernel
         @shellSocket.close()
         @ioSocket.close()
 
-        if @onlyConnect
+        if @kernelProcess?
+            @kernelProcess.kill 'SIGKILL'
+            fs.unlink @connectionFile
```
Comment (lgeiger) [inline lib/zmq-kernel.coffee]: Clean up the created connection file

**Comment by lgeiger** [comment]
Waiting for https://github.com/nteract/spawnteract/pull/13 to be merged.

**Comment by n-riesco** [comment]
WFM

**Comment by rgbkrk** [comment]
Yay! Check it out @jdfreder, @willingc, and @captainsafia!

**Comment by willingc** [comment]
@rgbkrk Nice!!!!

**Pull request merged**